### ISSUE TITLE
Create Table $unique is not working

### ISSUE THREAD

**Issue by kokhong20** [meteor-stream/meteor-postgres#239]
![screen shot 2015-04-26 at 2 41 10 pm](https://cloud.githubusercontent.com/assets/3205289/7336192/5f1c1048-ec22-11e4-9824-3aaa2841dc15.png)
In Todo List example, when i am creating table using the data types and options below. The $unique give me the above error. Any ideas why $unique is not working?

```javascript
var usersTable = {
    id: ['$number'],
    name: ['$string', '$notnull','$unique']
  };
  username.createTable(usersTable);
```

**Comment by barbbella** [comment]
Where is this in your code, client or server?

**Comment by agershun** [comment]
This is the AlaSQL issue. Current version (0.1.3) does not support UNIQUE column constraints. Sorry for that :(

The SQL syntax is fixed already in version 0.1.4, but it will require some time to add this functionality to INSERT and UPDATE statements.

**Comment by agershun** [comment]
I fixed this problem and added UNIQUE (as well as FOREIGN KEYs and CHECK constraints). This functionality will be available in version 0.1.4 of AlaSQL or now from ```develop``` branch. Could you try it?

**Comment by pdiniz13** [comment]
Ill test it in the next couple of days and let you know how it works out. Thank you for the update.
